### PR TITLE
Add grouping for tasks in AzureDevOps pipeline

### LINKED ISSUE #111: Add grouping for tasks in AzureDevOps pipeline
With the changes of #110 it is possible to use the ##[group] command.

This can be used to change the output from 
image
to
image

### PR DESCRIPTION
**Pull request by JuergenRB**
Closes #111

### CODE REVIEW

**Comment by nils-a** [comment]
@JuergenRB Thanks for the PR!
May I ask, what was the reasoning for implementing an `IExecutionStrategy` instead of doing this in the existing `AzurePipelinesEngine`? 

The `TeamCityEngine` provides "groups" very similar to what was done here (see #101), but that is all based on the existing events provided by `ICakeEngine`. Am I missing something?

**Comment by JuergenRB** [comment]
@nils-a 
My idea was not to have duplicate task titles in the log output. 
That's why I implemented it in ```IExecutionStrategy``` .

If you want, I can rebuild it so that it is solved in a consistent way.

**Comment by nils-a** [comment]
@JuergenRB yes, having the functionality consistent would be good.

**Comment by JuergenRB** [comment]
Changed
image

File: src/Cake.AzurePipelines.Module/Cake.AzurePipelines.Module.csproj [hunk 1]
```diff
@@ -7,6 +7,10 @@
     <IsPackable>false</IsPackable>
   </PropertyGroup>
 
+  <ItemGroup>
+    <Compile Remove="AzurePipelinesExecutionStrategy.cs" />
+  </ItemGroup>
+
```
Comment (nils-a) [inline src/Cake.AzurePipelines.Module/Cake.AzurePipelines.Module.csproj:13]: Could you remove this, since the file no longer exists?

**Review by nils-a — CHANGES_REQUESTED** [review]
LGTM in general. One small change, though.

**Review by nils-a — APPROVED** [review]
LGTM

**Pull request merged**